lib/containerhelpers/postgres.go

```diff
@@ -0,0 +1,26 @@
+package containerhelpers
```

Comment:
For what do we need this file?

Reply:
es wird ein docker-container für die unit tests erstellt mit der postgres db